pkg/aws_config_server/types.go

```diff
@@ -36,6 +37,20 @@ func (a *AWSConfig) GetAccounts() []AWSAccount {
 	return accounts
 }
 
+func (a *AWSConfig) GetRoleNames() []string {
+	set := map[string]bool{}
+	roleNames := []string{}
+
+	for _, profile := range a.Profiles {
+		roleName := strings.Split(profile.RoleARN, "role/")[1]
```

Comment:
rather than doing this split, should we add the role name to the profile struct further upstream?